### ISSUE TITLE
saveaspng option in Shiny

### ISSUE THREAD

**Issue by alhenry** [davidgohel/ggiraph#120]
Hi,

I am trying to create a Shiny app with the ggiraph package which allows users to download the static version of the plot generated with `girafe()`, but I'm struggling to get the `girafe_options (opts_toolbar (saveaspng =TRUE))` work within the app. The rendered plot correctly shows the toolbar, but the download button doesn't respond on click.

Is this an expected behaviour and if so, is there any work around for this? 

Thanks!

**Comment by alhenry** [comment]
Hi,

Thank you for your response. I am using Chrome. I did some more tests and managed to get the save function working with simpler example. In my Shiny app, I used several `reactive()` objects to create the dataset for plotting controlled by `isolate()` function, and then use the `patchwork` package to combine multiple ggplot objects. Would any of these change the behaviour of the save function?

Thanks again!
